src/contentmanagerside.ui

```diff
@@ -109,6 +106,9 @@
          </property>
         </widget>
        </item>
+       <item>
```

Comment:
This comment is about the commit message.
`Moved as per suggestion` is definitively not a good commit message.
It is nice to discard you about the choice made but it will not help us to understand the choice in few months.
The reason why you made this change is not "It has been suggested to me".
The suggestion come for a reason (and you have agree with it, else you would not do the change), it is better to directly write the reason.

Reply:
Done.